### PR TITLE
Add lines marking constant state and rate dependence

### PR DESCRIPTION
**Pull request by jrleeman**
Improves the figure by marking lines of constant a and rate dep. of friction. Addresses #16

### CODE REVIEW

File: rsfmodel/rsf.py [hunk 1]
```diff
@@ -194,12 +194,30 @@ def solve(self, **kwargs):
 
 def phasePlot(system):
```
Comment (dopplershift) [inline rsfmodel/rsf.py]: If you're sure of the fixed size here, feel free to stick with it. But here's a pattern I often use:

``` python
def phasePlot(system, fig=None):
    if fig is None:
        fig = plt.figure()

**Comment by jrleeman** [comment]
I had just put that size since it looked nice, but this is better. Should we also return the figure and/or axis objects?

I was a little worried about the way to draw the lines of constant a. This may be somewhat clunky.

**Comment by dopplershift** [comment]
Returning `fig` is probably a good idea. Could also take/return `ax` the same way if you wanted.

As far as the lines, I've done something similar with my skewT code. You could create a `LineCollection` object instead of calling `plot()`, but I'm not sure that really helps readability at all. For me it was more a matter of avoiding the `pyplot` interface.

**Comment by jrleeman** [comment]
I added the fig/axes returns. I think for now I'll leave the plot call, but we could revisit if we want to.

**Pull request merged**